Francisco_Trujillo/Assignments/multiply/multiply.py

```diff
@@ -0,0 +1,9 @@
+#Create a function that multiply elements of a List by 5
+def multiply(a,m):
+    b = []
+    for element in a:
+        b.append(element * m)
```

Comment:
well done. you may also want to check out / do research on Python List Comprehensions.